src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -632,4 +632,7 @@
   <data name="Apply_readonly_struct_preferences" xml:space="preserve">
     <value>Apply 'readonly struct' preferences</value>
   </data>
+  <data name="Selection_cannot_be_in_base_constructor_initialize" xml:space="preserve">
+    <value>Selection cannot be in base constructor initializer</value>
+  </data>
```

Comment:
do we have a test for extracting a method?

Reply:
I'll check. It should be valid for those cases.

Reply:
I added for extract method and found some bugs. Filed them as follow up and noted in the tests that are skipped